provider/transaction.go

```diff
@@ -24,3 +24,78 @@ type SendTransactionInput struct {
 	Address     string `json:"address"`
 	RawHexBytes string `json:"raw_hex_bytes"`
 }
+
+// GetAccountTransactionsOutput represents output for GetAccountTransactions request
+type GetAccountTransactionsOutput struct {
+	PageCount int            `json:"page_count"`
+	TotalTxs  int            `json:"total_txs"`
+	Txs       []*Transaction `json:"txs"`
+}
+
+// GetBlockTransactionsOutput represents output for GetBlockTransactions request
+type GetBlockTransactionsOutput struct {
+	PageCount int            `json:"page_count"`
+	TotalTxs  int            `json:"total_txs"`
+	Txs       []*Transaction `json:"txs"`
+}
+
+// Transaction represents a transaction in Pocket
+type Transaction struct {
+	Hash   string `json:"hash"`
+	Height int    `json:"height"`
+	Index  int    `json:"index"`
+	Proof  struct {
+		Data  string `json:"data"`
+		Proof struct {
+			Aunts    []string `json:"aunts"`
+			Index    int      `json:"index"`
+			LeafHash string   `json:"leaf_hash"`
+			Total    int      `json:"total"`
+		} `json:"proof"`
+		RootHash string `json:"root_hash"`
+	} `json:"proof"`
+	StdTx struct {
+		Entropy int64 `json:"entropy"`
+		Fee     []struct {
+			Amount string `json:"amount"`
+			Denom  string `json:"denom"`
+		} `json:"fee"`
+		Memo string `json:"memo"`
+		Msg  struct {
+			Type  string `json:"type"`
+			Value struct {
+				Amount      string `json:"amount"`
+				FromAddress string `json:"from_address"`
+				ToAddress   string `json:"to_address"`
+			} `json:"value"`
+		} `json:"msg"`
+		Signature struct {
+			PubKey    string `json:"pub_key"`
+			Signature string `json:"signature"`
+		} `json:"signature"`
+	} `json:"stdTx"`
+	Tx       string `json:"tx"`
+	TxResult struct {
```

Comment:
TxResult varies depending on the type of the transaction, this should be thread in a more general sense

Reply:
Do you propose using `interface{}` or can we be more specific?

Reply:
> TxResult varies depending on the type of the transaction, this should be thread in a more general sense

StdTx is what changes depending on the type of the transaction, TxResult is always the same

Reply:
So, let's give a general structure to that one instead if possible

Reply:
The `Msg` is the only thing that varies btw

Reply:
> The `Msg` is the only thing that varies btw

you're right, on that case `any` can be added to that in favour of `interface{}` as both mean the same and is more modern term

Reply:
Used `interface{}` because that's what I'm using everywhere on the SDK